packages/shared/src/components/search/SearchBarSuggestionList.tsx

```diff
@@ -86,4 +86,4 @@ export function SearchBarSuggestionList({
       ))}
     </div>
   );
-}
+}   
```

Comment:
We should not need the extra space.
```suggestion
}
```